lib/MusicBrainz/Server/Report/NotVariousArtists.pm

```diff
@@ -0,0 +1,47 @@
+package MusicBrainz::Server::Report::NotVariousArtists;
+use Moose;
+use MusicBrainz::Server::Constants qw( $VARTIST_ID );
+
+with 'MusicBrainz::Server::Report::RecordingReport',
+     'MusicBrainz::Server::Report::FilterForEditor::RecordingID';
+
+sub query {
+    "
+        SELECT
+            r.id AS recording_id,
+            row_number() OVER (ORDER BY r.artist_credit, r.name)
+        FROM recording r
+        JOIN artist_credit ac on ac.id = r.artist_credit
+        JOIN artist_credit_name acn on acn.artist_credit = r.artist_credit
+        WHERE acn.artist = $VARTIST_ID AND ac.name != 'Various Artists'
```

Comment:
Should be `acn.name`, right? Since we'd want to catch this even if the AC contains other artists, I'd think.

You don't need to join with `artist_credit` that way, either.